install/wrap-up.sh

```diff
@@ -3,7 +3,7 @@ if [[ "$MINIMIZE_DOWNTIME" ]]; then
 
   # Start the whole setup, except nginx and relay.
   $dc up -d --remove-orphans $($dc config --services | grep -v -E '^(nginx|relay)$')
-  $dc exec -T nginx service nginx reload
+  $dc restart nginx
```

Comment:
just restart the container, it's the same as trying to reload the container
I did not find a quick alpine way to reload nginx without installing one more package into it just for that one time

Reply:
This is the reason: https://github.com/getsentry/onpremise/pull/615#discussion_r477541160